src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -98,6 +101,16 @@ export class NavigationComponent implements OnInit, OnDestroy {
     );
   }
 
+  getFormattedClusterName(cluster: MultiCluster, shortenFsid: boolean): string {
+    if (shortenFsid) {
+      return `${cluster?.cluster_alias} (${cluster?.user}) - ${this.multiClusterService.shortenName(
+        cluster?.name
+      )}`;
+    } else {
+      return `${cluster?.cluster_alias} (${cluster?.user}) - ${cluster?.name}`;
```

Comment:
just add a validation if the cluster ID is added we can skip adding cluster again for now

Reply:
Thanks, updated the PR